code/game/gamemodes/revolution/revolution.dm

```diff
@@ -63,9 +63,6 @@
 	var/list/sec = get_living_sec()
 	var/weighted_score = min(max(round(heads.len - ((8 - sec.len) / 3)),1),max_headrevs)
 
-	if(weighted_score < 1)
-		weighted_score = 1
-

```

Comment:
It was a left over from the pull that introduced this bug. The max({formula},1) will do the same thing.